scalability_and_performance/recommended-host-practices.adoc

```diff
@@ -29,6 +29,12 @@ include::modules/increasing-aws-flavor-size.adoc[leveloffset=+2]
 
 include::modules/recommended-etcd-practices.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+[id="additional-resources_recommended-etcd-practices"]
+.Additional resources
+* link:https://github.com/peterducai/openshift-etcd-suite[etcd troubleshooting tools on {product-title}]
```

Comment:
Linking to documentation and resources hosted on GitHub pages outside of the `openshift` organization is not allowed. Occasionally, exceptions are made but those need to go our change management process. In most cases, the exceptions that are approved tend to be hosted in an official upstream repo. 

I suggest cutting this link from the current PR and revisiting it in a follow up ticket so you don't hold up the rest of the content.

Reply:
Done.